api/devices/schema.py

```diff
@@ -108,6 +108,28 @@ def mutate(self, info, device_id, **kwargs):
         return DeleteDevice(device=exact_device)
 
 
+class UpdateDeviceActivity(graphene.Mutation):
+    """Changes a device activity from online to offline"""
+    class Arguments:
+        device_id = graphene.Int(required=True)
+        activity = graphene.String()
+
+    device = graphene.Field(Devices)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, device_id, **kwargs):
+        query_device = Devices.get_query(info)
+        result = query_device.filter(DevicesModel.activity == "offline")
+        exact_device = result.filter(
+            DevicesModel.id == device_id
+        ).first()
+        if not exact_device:
```

Comment:
@Arusey, you could make the message a bit more specific. When the device is online and the user tries to turn the device online, the message `Device not found` is a bit confusing. You can make the message to be `Device not found or already turned online`.

Reply:
noted